# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/nload/noinit_val.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/form_field.cpp.org	Mon Jan 20 13:56:48 2003
+++ ./src/form_field.cpp	Mon Jan 20 13:58:23 2003
@@ -133,7 +133,7 @@
 }
 
 
-Form::Form( Slots* slots = 0 ) : m_slots( slots ), m_form(0), m_visible( false )
+Form::Form( Slots* slots ) : m_slots( slots ), m_form(0), m_visible( false )
 {
 	m_instances.push_back( this );
 }
--- ./src/form_field.h.org	Mon Jan 20 13:56:52 2003
+++ ./src/form_field.h	Mon Jan 20 13:58:36 2003
@@ -86,7 +86,7 @@
 		virtual void slot_fieldChanged( Field* field ) {}
 	};
 	
-	Form( Slots* slots = 0 );
+	Form( Slots* slots );
 	~Form();
 	
 	vector<Field *>& fields();
